test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua

```diff
@@ -0,0 +1,53 @@
+--  Requirement summary:
+--  [RegisterAppInterface] REJECT
+--  [RegisterAppInterface] RegisterAppInterface and HMILevel
+--
+--  Description:
+--  Check that App will be rejected with HMI type WEB_VIEW 
+--  when policy table is empty.
+--
+--  Used precondition
+--  SDL, HMI are running on system.
+--  Mobile device is connected to SDL.
+--
+--  Performed steps
+--  appID->RegisterAppInterface(params)
+--
+--  Expected behavior:
+--     Policy table is empty.
+--     SDL->appID: REJECT, success:"false":RegisterAppInterface()
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared Libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
```

Comment:
```suggestion
--[[ Test Configuration ]]
config.defaultMobileAdapterType = "WS"
common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
```
Update test to check webEngine app over the WebSocket connection

Reply:
@VjKlepikov [done](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/4/commits/0f412587275cf5245150b036bc28189967421e9c#diff-ff5652b3ded34af081249da10ca6203fR23)